disputer/test/Disputer.js

```diff
@@ -45,7 +45,7 @@ contract("Disputer.js", function(accounts) {
   const unreachableDeadline = MAX_UINT_VAL;
 
   before(async function() {
-    collateralToken = await Token.new("UMA", "UMA", 18, { from: contractCreator });
+    collateralToken = await Token.new("DAI", "DAI", 18, { from: contractCreator });
```

Comment:
unrelated change: collateral should not be called `UMA`

Reply:
_everything_ should be called `UMA`